OneSignalSDK/app/src/main/java/com/onesignal/MainActivity.java

```diff
@@ -268,6 +268,18 @@ public void onUnsubscribeClicked(View v) {
 
    public void onSendTagsClicked(View v) {
       try {
+         OneSignal.setExternalUserId("mikechoch_" + sendTagsCounter, new OneSignal.ExternalUserIdUpdateHandler() {
```

Comment:
Can the id be a generic one? Although it's a demo app, It should exclude reference to its developers, what do you think?

Reply:
I did this lazy for testing, there is a text field now haha